uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -578,7 +578,23 @@ sub move_minc {
     ### figure out where to put the files ######################
     ############################################################
     $dir = $this->which_directory($subjectIDsref,$data_dir);
-    `mkdir -p -m 770 $dir/native`;
+
```

Comment:
I know the next changes are not pretty but the line we had before, albeit compact, only sets the permissions properly on the last directory only (native in this case). the -p does not propagate the permissions set in -m option to the earlier subdirectories (hence CandID/ and /VisitLabel/ and mri/ all had permissions other than 770).
Below I invoke mkdir with -m option for each of the directories in the hierarchy